entries/o/office.com.json

```
@@ -1,6 +1,11 @@
 {
-  "Office 365": {
-    "domain": "office.microsoft.com",
+  "Microsoft 365": {
+    "domain": "office.com",
+    "additional-domains": [
```

Since the login happens on that domain, this entry should help password managers and browser extensions.
```suggestion
    "additional-domains": [
      "login.microsoftonline.com",
```